subscriptionform/settings/base.py

```diff
@@ -37,7 +37,6 @@
     'django.contrib.staticfiles',
 
     # Project apps
-    'sports',
```

Comment:
abc zegt over het algemeen dat de s voor de u komt, wat is je reden om het om te gooien?